runtime/caml/domain_state.tbl

```diff
@@ -104,6 +104,8 @@ DOMAIN_STATE(struct pool**, pools_to_rescan)
 DOMAIN_STATE(int, pools_to_rescan_len)
 DOMAIN_STATE(int, pools_to_rescan_count)
 
+DOMAIN_STATE(value, tls)
```

Comment:
Perhaps `tls_root` would a better name for this.

Reply:
Yes, changed it.